dist/index.js

```diff
@@ -310,10 +316,14 @@ function runAction() {
             core.info(`Starting the execution of the pipeline with id ${executionGraphId}, check the pipeline details: ${getDownloadVibPublicUrl()}/v1/execution-graphs/${executionGraphId}`);
             // Now wait until pipeline ends or times out
             let executionGraph = yield getExecutionGraph(executionGraphId);
+            let pipelineDuration = getNumberInput("max-pipeline-duration");
```

Comment:
Just realized we are missing a `* 1000` here as the input value will be in seconds.

Reply:
done